set.mm

```diff
@@ -389540,7 +389540,15 @@ then any point of the half line ( ` R ` ` A ` ) also lies opposite to
        segment and each angle (6 parts).  This is often the actual textbook
        definition of triangle congruence, see for example
 ~ https://en.wikipedia.org/wiki/Congruence_(geometry)#Congruence_of_triangles
-       (Contributed by Thierry Arnoux, 18-Jan-2023.) $)
+       With this definition, the "SSS" congruence theorem has an additional
+       part, namely, that triangle congruence implies congruence of the length
+       of each sides.  Because our development of elementary geometry strives to
```

Comment:
congruence of length --> do you mean "equality of lengths"

And by the way: isn't equality of lengths precisely the definition of congruence ?

Reply:
To clarify:
* Definition: two sets of points A and B are congruent iff there exists a bijection f: A >->> B such that for all x, y in A, one has dist(x, y) = dist(f(x), f(y)).
* In particular, two segments are congruent iff their lengths are equal.